### PR TITLE
Add consistent ARM64 build image process

### PR DESCRIPTION
**Pull request by rdimitrov**
Signed-off-by: Radoslav Dimitrov <rdimitrow@gmail.com>

## Checklist:

I have:

- [ ] updated the documentation if required
- [x] read the [CONTRIBUTION](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md) guide
- [x] signed-off my commits with `git commit -s`
- [ ] added unit tests

### CODE REVIEW

File: Dockerfile.arm64 [hunk 1]
```diff
@@ -0,0 +1,25 @@
+FROM golang:1.10.4 AS builder
```
Comment (LucasRoesler) [inline Dockerfile.arm64]: The default `Dockerfile` uses golang alpine, can we use that here too?

Comment (rdimitrov) [reply Dockerfile.arm64]: For ARM64 it failed during the build process, I believe it was due to some dependency issue. 

If there's a problem to use this one, I can try to build the ARM64 image with `alpine` and resolve the issue?

Comment (rdimitrov) [reply Dockerfile.arm64]: Managed to resolve the dependency by adding the `build-base` package. :+1:

**Comment by alexellis** [comment]
Thanks @rdimitrov 👍

**Pull request merged**